### PR TITLE
Docs: Add Alerting export to what's new

### PR DESCRIPTION
**Pull request by yuri-tceretian**
Adds alerting export to HCL introduced recently. 

Related:
https://github.com/grafana/grafana/pull/75796
https://github.com/grafana/grafana/pull/75849
https://github.com/grafana/grafana/pull/76411
https://github.com/grafana/grafana/pull/73166
https://github.com/grafana/grafana/pull/76016

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 1]
```diff
@@ -18,6 +18,16 @@ weight: -37
...
 
+## Export Alert rules and notification resources to Terraform
+<!-- Yuri Tseretyan -->
+<!-- OSS, Enterprise -->
+
+October 30, 2023
+
+_Generally available in Grafana Cloud_
+
+Provides a way to export alerting resources such as rules, contact points, notification policies as a Terraform resources. A new mode "Modify export" for alert rules provides a convenient way of editing provisioned alert rules and export the modified version.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:29]:
```suggestion
This feature provides a way to export alerting resources such as rules, contact points, and notification policies as Terraform resources. A new "Modify export" mode for alert rules provides a convenient way of editing provisioned alert rules and exporting the modified version.   
```

**Review by imatwawana — APPROVED** [review]
Made some wording edits. Just wondering, is this intended for the Grafana 10.2 release or 10.3?

**Comment by yuri-tceretian** [comment]
This is intended for 10.2

**Comment by imatwawana** [comment]
> This is intended for 10.2

Okay I'll add it to that 10.2 What's new.

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 2]
```diff
@@ -18,6 +18,17 @@ weight: -37
 
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
 
+## Export Alert rules and notification resources to Terraform
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:21]:
```suggestion
## Export alert rules and notification resources to Terraform
```

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 3]
```diff
@@ -18,6 +18,17 @@ weight: -37
...
+
+<!-- Yuri Tseretyan -->
+<!-- OSS, Enterprise -->
+
+October 30, 2023
+
+_Generally available in Grafana Cloud_
+
+This feature provides a way to export alerting resources such as rules, contact points, and notification policies as Terraform resources. A new "Modify export" mode for alert rules provides a convenient way of editing provisioned alert rules and exporting the modified version.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:30]:
```suggestion
This feature provides a way to export Alerting resources such as rules, contact points, and notification policies as Terraform resources. A new "Modify export" mode for alert rules provides a convenient way of editing provisioned alert rules and exporting the modified version.
```

**Review by imatwawana — COMMENTED** [review]
Two small nits that I missed on the first pass. To my knowledge, alert rules are lower case, but Alerting (the feature) is always upper case.

**Pull request merged**